src/modules/functions.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-use-before-define */
+import {
+  toDoContainer,
+} from './variable.js';
+
+let arr = [];
+
+const appendToParent = (elem) => {
+  toDoContainer.innerHTML += `<div class="activity-container">
+    <input type="checkbox" class="checkBox">
+    <li class =" hidden" >${arr[arr.indexOf(elem)].description}</li>
+    <input type="text" id="edit-to-do" readonly class ="item" value ='${arr[arr.indexOf(elem)].description}' >
+    <i class="fa-solid fa-ellipsis-vertical drag"></i>
+    <i class="fa-regular fa-trash-can hidden delete"></i>
+    </div>`;
+};
+
+const add = (input) => {
+  arr.push({
+    description: input.value,
+    completed: false,
+    index: arr.length,
+  });
+};
```

Comment:
- [ ] When a task is added to the local storage its index property should be equal to its position on the array. Now when I add two tasks, for example, the index properties are 0 and 1, instead of 1 and 2. Kindly rectify that.

  ![rrp](https://user-images.githubusercontent.com/54848363/193008057-061ecf8c-2160-41a0-879c-75ea33198942.jpg)